builder/builder.go

```diff
@@ -343,15 +344,17 @@ func buildWhereCondition(mapSet *whereMapSet) ([]Comparable, func(), error) {
 	return cpArr, release, nil
 }
 
-func convertWhereMapToWhereMapSlice(where map[string]interface{}) (map[string][]interface{}, error) {
+func convertWhereMapToWhereMapSlice(where map[string]interface{}, op string) (map[string][]interface{}, error) {
 	result := make(map[string][]interface{})
 	for key, val := range where {
 		vals, ok := convertInterfaceToMap(val)
 		if !ok {
-			return nil, errWhereInType
+			errMsg := fmt.Sprintf(errWhereInterfaceSliceType, op)
+			return nil, errors.New(errMsg)
```

Comment:
`fmt.Errof(fmt, xx)` would be better

Reply:
Done [enhancement](https://github.com/didi/gendry/pull/62/commits/b957667a07beddaccbf4ca320bb506e91b74ab44).